recipes-multimedia/alsa/alsa-utils/90-alsa-restore.rules.in

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
+
+# When a sound device is detected, restore the volume settings
+  KERNEL=="controlC[0-9]*",  NAME="snd/%k", ACTION=="add", RUN+="@@UDEV_DIR@@/soundconfig %k"
```

Comment:
Please look at @@UDEV_DIR@@ it is more generic. I hope it is ok. The one on libreelec doesn't have explicit path but rather RUN+="soundconfig %k" but it doesn't work for me.